### ISSUE TITLE
A11y_AzureDataStudio_Newdeployment-AzureDatabaseforMySQL_ScreenReader: ORCA screen reader is announcing unnecessary whole dialog information when we select any list item from the "Version" dropdown.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22692]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG1.3.1; #Linux; #Element:EditField; #ScreenReader; #ORCA; 

### Actual Result:​
ORCA screen reader is announcing unnecessary whole dialog information when we select any list item from the "Version" dropdown.

### Expected Result:​
ORCA screen reader should not announce unnecessary whole dialog information when we select any list item from the "Version" dropdown.

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to ellipses '...' and select "New Deployment".
4. TAB to Deployment options. TAB "Azure Database For MySQL" card and hit ENTER key.
5. Start the ORCA screen reader. TAB to "Version" dropdown and select any list item from the list. 
6. Observe that ORCA screen reader is announcing unnecessary whole dialog information when we select any list item from the "Version" dropdown.

### User Impact:
Screen reader users will get confused about the announcement as its announcing whole dialog info.

### Attachment:

[Video 3-30 at 12.37 (4).webm](https://user-images.githubusercontent.com/90901519/228815004-6b006599-711b-402d-ab51-77a2eb215c40.webm)

**Comment by alanrenmsft** [comment]
this is also a typical screen reader issue, we don't control how the screen reader decides to read out the content and events, we have set all the necessary a11y attributes, how does it announce is really out of our control, please close.

@PoojaNamde

**Comment by PoojaNamde** [comment]
@alanrenmsft 

Could you please provide the sample app to check the behavior of screen reader.
